<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemPreferences</class>
 <widget class="QWidget" name="SystemPreferences">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="showSharedPrintersCB">
     <property name="text">
      <string comment="@option:check">Show printers shared by other systems</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="shareConnectedPrintersCB">
     <property name="text">
      <string comment="@option:check">Share printers connected to this system</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="allowFromInternetCB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string comment="@option:check">Allow printing from the Internet</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="allowRemoteAdminCB">
     <property name="text">
      <string comment="@option:check">Allow remote administration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="useKRBAuthCB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string comment="@option:check">Use Kerberos authentication</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="allowUsrCancelCB">
     <property name="text">
      <string comment="@option:check">Allow users to cancel any job (not just their own)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>501</width>
       <height>202</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shareConnectedPrintersCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>allowFromInternetCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
